dspace/config/local.cfg.EXAMPLE

```diff
@@ -244,3 +244,8 @@ db.schema = public
 
 # Maximum size of a multipart request (i.e. max total size of all files in one request)
 #spring.servlet.multipart.max-request-size = 512MB
+
+#################################################
+# SOLR Path configuration  #
+#################################################
+solr.home = /opt/bin/solr/solr-8.11.1
```

Comment:
I'd recommend we change this to be `solr.configSetBaseDir` as the directory we need is the base directory of the Configsets (and this directory is configurable, so it may not always be `[solr.home]/server/solr/configsets`, which you've assumed it is)

See the Solr docs here: https://solr.apache.org/guide/8_4/config-sets.html#configsets-in-standalone-mode

This `solr.configSetBaseDir` setting also needs to be added to dspace.cfg and have some basic comments documenting the setting.  It should probably be listed alongside the other Solr settings here: https://github.com/DSpace/DSpace/blob/main/dspace/config/dspace.cfg#L43-L67

By default, we could set it like:
```
# Specify Solr configsets directory. Defaults to a directory based on the SOLR_HOME
# environment variable, but you may need to modify it based on your Solr setup.
# This is not a required field, as it is only used for the './dspace solr-reindex-statistics'  command
solr.configSetBaseDir = ${env:SOLR_HOME}/configsets
```

I think that'd be a better solution here overall than making assumptions about the location of the configsets.